### PR TITLE
Perl updates

### PR DESCRIPTION
**Pull request by peterbud**
perl-Algorithm-Diff: Update to 1.201
perl-ExtUtils-MakeMaker: Update to 7.58
perl-JSON: Update to 4.03
perl-IO-Socket-IP: Update to 0.41
perl-IO-Socket-SSL: Update to 2.069
perl-Net-SSLeay: Update to 1.90
perl-Path-Tiny: Update to 0.116
perl-Role-Tiny: Update to 2.002004
perl-Test-Fatal: Update to 0.016
perl-Importer: Update to 0.026
perl-XML-LibXML: Update to 2.0206
perl-List-MoreUtils: Update to 0.430
perl-Test-Simple: Update to 1.302183
perl-YAML-Syck: Update to 1.34
perl-Alien-Build: Update to 2.38
perl-Alien-Libxml2: Update to 0.17
perl-Params-Util: Update to 1.102
perl-Test2-Suite: Update to 0.000139
perl-HTTP-Message: Update to 6.27
perl-perl-HTTP-Cookies: Update to 6.10
perl-libwww: Update to 6.52
perl-Net-HTTP: Update to 6.20
perl-URI: Update to 5.07

### CODE REVIEW

File: perl-XML-LibXML/PKGBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 _realname=XML-LibXML
 pkgname=perl-${_realname}
-pkgver=2.0205
+pkgver=2.0206
 pkgrel=3
```
Comment (Biswa96) [inline perl-XML-LibXML/PKGBUILD:6]: Reset `pkgrel` value to 1.

Comment (peterbud) [reply perl-XML-LibXML/PKGBUILD]: Thanks for spotting, done

**Comment by Biswa96** [comment]
Is this ready to merge?

**Comment by lazka** [comment]
I'll have a look tomorrow, but feel free to merge if you think it's ok.

**Comment by peterbud** [comment]
From my POV it's ready to merge.

**Comment by piotrvv** [comment]
Please upgrade perl version to 5.32.1 with my changes from attached file.
[PKGBUILD.zip](https://github.com/msys2/MSYS2-packages/files/5931188/PKGBUILD.zip)

**Comment by lazka** [comment]
> Please upgrade perl version to 5.32.1 with my changes from attached file.

Please open a PR

**Pull request merged**